.env.prod

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+UNAME_INFO=$(uname -a)
```

Comment:
uncomment check_docker_command and other commands and use this var there

Reply:
Fixed